emailapi/mailsend.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+const nodeMailer = require('nodemailer');
+const file= require('./sendNotification.js');
+
+function senderFunction(mailOptions){
+	var transporter = nodeMailer.createTransport({
+	service: 'gmail',

```

Comment:
code indentation is incorrect

Reply:
I have done it in my file but changes are not reflecting over here.